README.md

```diff
@@ -1,33 +1,31 @@
-# [project name]
+# Memory-bias for errors dataset
```

Comment:
I think it will look "nicest" across all lab repos if we ultimately just match the repo name here for consistency.  Are you ok with that?  So:
memory-for-error-dataset